src/arrayMethodJoin.js

```diff
@@ -4,8 +4,24 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
+  [].__proto__.join2 = function(separator = ',') {
     // write code here
+    let finalStr = '';
+
+    for (let i = 0; i < this.length; i++) {
+      if (this[i] === null || this[i] === undefined) {
+        finalStr += `${separator}`;
```

Comment:
```suggestion
        finalStr += separator;
```
Можна просто використати змінну, а не конвертувати її в рядок перед цим